### PR TITLE
Integrations: Add Prometheus, Grafana and Argo

### LINKED ISSUE #2986: [Meshery] Integrations: Add Prometheus, Grafana and Argo
**Current Behavior**

The Meshery has three more upcoming integration which has not been added to the [Integration Collection](https://layer5.io/service-mesh-management/meshery/integrations) page. 

**Desired Behavior**

- Prometheus, Grafana and Argo should be on the [Integration Collection](https://layer5.io/service-mesh-management/meshery/integrations) Page.
- These images should be an SVG file.

---

<h4>Contributor Resources</h4>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by abhiyant-10**
Signed-off-by: abhiyant-10 <abhigwalani@gmail.com>

**Description**

This PR fixes #2986

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/assets/data/integrations/IntegrationList.js [hunk 1]
```diff
@@ -196,6 +199,24 @@ const IntegrationList = [
     status: "inProduction",
     category: "Service Mesh",
   },
+  {
+    name: "Prometheus",
+    img: prometheus,
+    status: "inProduction",
```
Comment (sahilsuman933) [inline src/assets/data/integrations/IntegrationList.js:205]: All these integrations are in the in-progress state to change the status to InProgress.

**Review by sahilsuman933 — CHANGES_REQUESTED** [review]
You don't need to add both SVG and png files. SVG is enough!

**Comment by sahilsuman933** [comment]
Don't edit the package.lock.json file.

**Comment by sahilsuman933** [comment]
Also, change the Prometheus SVG to this one
prometheus

**Comment by l5io** [comment]
🚀 Preview for commit ea7e396e42c67e3e98ca5e36f68197f37ea10f42 at: https://62ef81a88e24d669ead926c8--layer5.netlify.app

**Pull request closed without merging**